autoendpoint/src/extractors/subscription.rs

```diff
@@ -54,6 +55,15 @@ impl Default for VapidClaims {
     }
 }
 
+impl fmt::Display for VapidClaims {
```

Comment:
Technically not needed for this release, and I'm fine pulling it, but it's still a useful thing to have for debugging and development.

Reply:
Since this isn't even used by the new log statement (right?) it's fine.

Reply:
No, it's not used by the new log statement. I did use it for some internal things while I was working on stuff and realized it'd be useful to leave.